firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift

```diff
@@ -37,6 +37,53 @@ class CreditCardsTests: BaseTestCase {
         }
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306978
+    // SmokeTest
+    func testDeleteButtonFromEditCard() {
+        navigator.nowAt(NewTabScreen)
+        navigator.goto(CreditCardsSettings)
+        unlockLoginsView()
+        mozWaitForElementToExist(app.staticTexts[creditCardsStaticTexts.AutoFillCreditCard.autoFillCreditCards])
+        app.buttons[creditCardsStaticTexts.AutoFillCreditCard.addCard].tap()
+        addCreditCard(name: "Test", cardNumber: "2720994326581252", expirationDate: "0540")
+        // Tap on a saved card
+        mozWaitForElementToExist(app.staticTexts[creditCardsStaticTexts.AutoFillCreditCard.autoFillCreditCards])
+        app.tables.cells.element(boundBy: 1).tap()
+        // The "View card" page is displayed with all the details of the card
+        mozWaitForElementToExist(app.navigationBars[creditCardsStaticTexts.ViewCreditCard.viewCard])
+        XCTAssertTrue(app.tables.cells.element(boundBy: 1).staticTexts.elementContainingText("1252").exists)
+        let cardDetails = ["Test", "05 / 40"]
+        for i in cardDetails {
+            XCTAssertTrue(app.textFields[i].exists, "\(i) does not exists")
```

Comment:
I have seen this line failing in previous execution, did something change in the final implementation? 

```
testDeleteButtonFromEditCard()

XCUITests / CreditCardsTests
	17.5stestDeleteButtonFromEditCard()

XCUITests / CreditCardsTests
/Users/vagrant/git/firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift:56 - XCTAssertTrue failed - ••••••••••••1252 does not exists
/Users/vagrant/git/firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift:56 - XCTAssertTrue failed - ••••••••••••1252 does not exists
```

https://app.bitrise.io/build/a31acb27-26cb-4343-a47a-c377ba601f44

Reply:
Nothing has changed. Not sure why on our checks it cannot see the entire string, i just validated that the last digits from the card are visible.

Reply:
Just wanted to make sure the test is going to be rock solid to run on Bitrise

Reply:
This test should be rock solid now, but i will keep an eye on it. The first one added on credit card has never failed since it has been merged.